front_end/server/lib/routes/servers.js

```diff
@@ -1,4 +1,19 @@
 const express = require('express');
 const router = express.Router();
 const Server = require('../models/server');
-const bodyParser = require('body-parser').json();
\ No newline at end of file
+const bodyParser = require('body-parser').json();
+
+router
+  .get('/', (req, res, next) => {
+    Server.find()
+      .lean()
+      .then(servers => res.send(servers))
+      .catch(next);
+  })
+  .post('/', bodyParser, (req, res, next) => {
+    new Server(req.body).save()
```

Comment:
nah I think new Server actually does make sense, agent is just the process.  I was confused because I thought newServer was like, an http server that was created just to handle a post request lol. I will say, the schema for newServer actually looks more like a single time series data point.  Server should probably include like, IP address + some unique identifier + hardware stats.  and a separate schema for data points that reference the server (not sure how relational stuff works in this case)